src/components/wallets/WalletAccordion.tsx

```diff
@@ -127,13 +138,13 @@ export const WalletAccordion = ({ wallet }: WalletAccordionProps) => {
                 </TooltipIcon>
               </DetailSummaryLine>
               <DetailSummaryLine>
-                <Typography color="grey700" variant="subhead">
+                <Typography color={isWalletActive ? 'grey700' : 'grey600'} variant="subhead">
```

Comment:
SOrry I Can't comment and things that has been done already
 line 104 + 111, can you add `noWrap` to typography (it's to add an ellipsis)
Also in SummaryInfos css: 
`min-width: 20px` (so the ellipsis will work when we shring the screen)

Reply:
Done